pkg/cmd/account/gcp/create/create.go

```diff
@@ -196,13 +198,13 @@ func promptMissing(opts *CreateOptions) error {
 	}
 
 	if opts.Environments.Value == nil {
-		environmentIDs, err := selectors.EnvironmentsMultiSelect(opts.Ask, opts.Octopus,
+		envs, err := selectors.EnvironmentsMultiSelect(opts.Ask, opts.Octopus,
```

Comment:
attention @domenicsim1 this kind of refactoring is why I was asking for at least one or two basic tests over the create account stuff.